tools/qa/trapdoor_pydocstyle.py

```diff
@@ -65,9 +65,10 @@ def get_stats(self, config):
         print 'USING              : pydocstyle', version
 
         # Call pydocstyle in the directories containing Python code. All files will be
-        # checked, including test files. Missing tests are ignored (D103) because they are
-        # already detected by PyLint in a better way.
-        command = ['pydocstyle', '--match=.*\\.py', '--add-ignore=D103'] + \
+        # checked, including test files. Missing docstrings are ignored because they are
+        # detected by PyLint in a better way.
+        command = ['pydocstyle', '--match=.*\\.py',
+                   '--add-ignore=D100,D101,D102,D103,D104,D105'] + \
```

Comment:
PyLint also gives messages for missing docstrings and allows for better control over which methods must have one. (Pydocstyle checks all or nothing.)

In HORTON we don't do docstrings for unit test functions (beginning with `test_` in modules beginning with `test_`, because that makes the nosetests output useless.